doc/manual/migrating.rst

```diff
@@ -0,0 +1,45 @@
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|Filter data       |EEG = pop_eegfiltnew(EEG, lfreq, hfreq);                     | raw.filter(lfreq, hfreq)                           |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|Run ICA           |EEG = pop_runica(EEG);                                       | ica.fit(raw)                                       |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|Epoching data     || event_id = {'cond1', 'cond2'};                             || events = mne.find_events(raw)                     |
+|                  || Epochs = pop_epochs(EEG, event_id, [tmin, tmax]) ;         || event_id = dict(cond1=32, cond2=64)               |
+|                  ||                                                            || epochs = Epochs(raw, events, event_id, tmin, tmax)|
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|Selecting epochs  |Epochs = pop_epochs(EEG_epochs, {cond2});                    | epochs[cond2]                                      |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|ERP butterfly plot|pop_timtopo(EEG_epochs, ...);                                | evoked.plot()                                      |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+|Contrast ERPs     |pop_compareerps(EEG_epochs1, EEG_epochs2);                   | (evoked1 - evoked2).plot()                         |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+||                 ||                                                            || raw.save(fname)                                   |
+|| Save data       || EEG = pop_saveset(EEG, fname);                             || epochs.save(fname)                                |
+||                 ||                                                            || evoked.save(fname)                                |
++------------------+-------------------------------------------------------------+----------------------------------------------------+
+
+Note that MNE has functions to read a variety of file formats, not just :func:`mne.io.Raw`. The interested user is directed to the :ref:`IO documentation <ch_convert>`.
+
+Pitfalls
+--------
+
+* Python operations are natively in place. To explicitly ask for a copy like in Matlab, use the boolean argument ``copy`` provided in MNE functions. 
```

Comment:
Operations? You mean methods? A lot of our functions are not in place (for to us, but probably not a beginner, obvious reasons). Reformulate.